rockcraft.yaml

```diff
@@ -63,5 +63,5 @@ parts:
             fields+=('${Source:Version}')
             printf -v dpkg_ops '%s,' "${fields[@]}"
             dpkg-query -W -f "${dpkg_ops}\n" \
-            --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ \
+            --root=${CRAFT_PROJECT_DIR}/../parts/postgresql-snap/layer/ \
```

Comment:
This won't work if we install packages in different parts. Then we'll have to merge the manifest manually.

Reply:
I would like to participate some tech talk/training about snap layers.